core/domain/email_validators.py

```diff
@@ -0,0 +1,259 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validators for email models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+
+from core.domain import base_model_validators
+from core.platform import models
+
+(
+    base_models, email_models, feedback_models,
+    suggestion_models, user_models
+) = models.Registry.import_models([
+    models.NAMES.base_model, models.NAMES.email, models.NAMES.feedback,
+    models.NAMES.suggestion, models.NAMES.user
+
+])
+
+USER_ID_REGEX = 'uid_[a-z]{32}'
```

Comment:
Use the one defined in the base_model_validators

Reply:
I have used the same as advised by you. Kindly let me know if I have done something wrong.

Reply:
What @DubeySandeep means here is that instead of redefining the `USER_ID_REGEX` here you should just use `base_model_validators.USER_ID_REGEX`.

Reply:
Done

Reply:
Thanks for the explanation @vojtechjelinek and sorry for the confusion @krishita30j.